.gitignore

```diff
@@ -26,6 +26,7 @@ faces-config.NavData
 *.DS_Store
 .java-version
 .gradle
+.worktree
```

Comment:
What is this ? :sweat_smile:

Reply:
It's the directory I like to use to put my git worktrees. See `git worktree --help`